py3status/modules/spaceapi.py

```diff
@@ -23,12 +25,15 @@
 import datetime
 import json
 import urllib.request
+import shlex
+import subprocess
 
 
 class Py3status:
     """
     """
     # available configuration parameters
+    button_action = 3
```

Comment:
would `button_open` or some other name be more descriptive?  open doesn't feel perfect but action just feels too vague.